include_all_in_one/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# This script will create the amalgamated library into 
+#    include_all_in_one/include/fplus/fplus.hpp
+#
+# Notes:
+# - include_all_in_one/include/fplus/fplus.hpp is part of the source repository, so that using FunctionalPlus 
+#   online with tools such as Compiler Explorer is easy. Just write:
+#     #include "https://raw.githubusercontent.com/Dobiasd/FunctionalPlus/master/include_all_in_one/include/fplus/fplus.hpp"
+# - this file will be updated automatically at each build, so that library developpers do not forget 
+#   to commit it (ideally once per PR)
+find_package (Python COMPONENTS Interpreter QUIET)
+if (Python_FOUND)
+    add_custom_target(
+        include_all_in_one
+        ALL
+        COMMAND ${Python_EXECUTABLE} ${CMAKE_CURRENT_LIST_DIR}/make_all_in_one.py
```

Comment:
Please always quote arguments that have variable expansion in them. Unquoted variable expansion can result in unintended list expansion, but this is also to be consistent with the rest of the files.

Once this is inlined into the root CML, please use `PROJECT_SOURCE_DIR` instead of `CMAKE_CURRENT_LIST_DIR` for consistency.

Since this command generates a file that is known ahead of time, the `BYPRODUCTS` argument should be used to indicate as such.